src/shared/components/BeerCard/BeerCard.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import classNames from 'classnames';
 import Button from '../Button/Button';
-import './BeerCard.css';
+import './styles/BeerCard.scss';
 
 export default function BeerCard({ id, image, title, tagline, description, classNames: classes }) {
```

Comment:
It looks like we don't pass classNames prop to BeerCard component from BeersList. Please check it and remove unnecessary props